caddy/caddymain/run.go

```diff
@@ -409,13 +415,70 @@ func initTelemetry() error {
 	return nil
 }
 
+// LoadEnvFromFile loads additional envs if file provided and exists
+//Envs in file should be in KEY=VALUE format
+func LoadEnvFromFile(envFile string) error {
+	if envFile == "" {
+		return nil
+	}
+
+	file, err := os.Open(envFile)
+
+	if err != nil {
```

Comment:
Move error check up by opening the file (no empty line)

Reply:
Vscode linter actually yells at me on this one:)